backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -399,8 +399,8 @@ object VeloxBackendSettings extends BackendSettingsApi {
             case _ =>
           }
           windowExpression.windowFunction match {
-            case _: RowNumber | _: Rank | _: CumeDist | _: DenseRank | _: PercentRank |
-                _: NthValue | _: NTile =>
+            case _: RowNumber | _: Rank | _: CumeDist | _: DenseRank | _: PercentRank | _: NTile =>
+            case nv: NthValue if !nv.input.foldable =>
```

Comment:
Just for brevity. Maybe, we can use:
```
case _ @NthValue(input, _, _) if !input.foldable =>
```

Ditto for Lead/Lag.

Reply:
This method is more friendly to changes in the number of parameters that may exist in multi-version support. Generally, we will not change the parameter names. What do you think?

Reply:
@zml1206, it makes sense. Thanks!